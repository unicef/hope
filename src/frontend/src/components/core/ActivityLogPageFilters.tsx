import ViewModuleRoundedIcon from '@mui/icons-material/ViewModuleRounded';
import { Grid, MenuItem } from '@mui/material';
import * as React from 'react';
import { useTranslation } from 'react-i18next';
import { useLocation, useNavigate } from 'react-router-dom';
import { AssigneeAutocomplete } from '@shared/autocompletes/AssigneeAutocomplete';
import { createHandleApplyFilterChange } from '@utils/utils';
import { FiltersSection } from './FiltersSection';
import { SearchTextField } from './SearchTextField';
import { SelectFilter } from './SelectFilter';

interface ActivityLogPageFiltersProps {
  filter;
  setFilter: (filter) => void;
  initialFilter;
  appliedFilter;
  setAppliedFilter: (filter) => void;
}
export function ActivityLogPageFilters({
  filter,
  setFilter,
  initialFilter,
  appliedFilter,
  setAppliedFilter,
}: ActivityLogPageFiltersProps): React.ReactElement {
  const { t } = useTranslation();
  const navigate = useNavigate();
  const location = useLocation();

  const { handleFilterChange, applyFilterChanges, clearFilter } =
    createHandleApplyFilterChange(
      initialFilter,
      navigate,
      location,
      filter,
      setFilter,
      appliedFilter,
      setAppliedFilter,
    );

  const handleApplyFilter = (): void => {
    applyFilterChanges();
  };

  const handleClearFilter = (): void => {
    clearFilter();
  };

  const modules = {
    program: 'Programme',
    household: 'Household',
    individual: 'Individual',
    grievanceticket: 'Grievance Tickets',
    paymentverificationplan: 'Payment Plan Payment Verification',
    targetpopulation: 'Target Population',
    registrationdataimport: 'Registration Data Import',
  };
  return (
    <FiltersSection
      clearHandler={handleClearFilter}
      applyHandler={handleApplyFilter}
    >
      <Grid container alignItems="center" spacing={3}>
        <Grid item xs={3}>
          <SearchTextField
            label={t('Search')}
            value={filter.search}
            onChange={(e) => handleFilterChange('search', e.target.value)}
            data-cy="filters-search"
          />
        </Grid>
        <Grid item xs={3}>
          <SelectFilter
            onChange={(e) => handleFilterChange('module', e.target.value)}
            label={t('Module')}
            value={filter.module}
            icon={<ViewModuleRoundedIcon />}
            SelectDisplayProps={{
              'data-cy': 'filters-residence-status',
            }}
            MenuProps={{
              'data-cy': 'filters-residence-status-options',
            }}
          >
            {Object.entries(modules)
              .sort()
              .map(([key, value]) => (
                <MenuItem key={key} value={key}>
                  {value}
                </MenuItem>
              ))}
          </SelectFilter>
        </Grid>
        <Grid item xs={3}>
          <AssigneeAutocomplete
            label={t('User')}
            filter={filter}
            name="userId"
            value={filter.userId}
            setFilter={setFilter}
            initialFilter={initialFilter}
            appliedFilter={appliedFilter}
            setAppliedFilter={setAppliedFilter}
          />
        </Grid>
      </Grid>
    </FiltersSection>
  );
}
