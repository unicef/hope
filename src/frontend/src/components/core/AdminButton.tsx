import ArrowCircleRightIcon from '@mui/icons-material/ArrowCircleRight';
import * as React from 'react';


interface AdminButtonProps {
    adminUrl: string;
    [key: string]: any;
}

export const AdminButton: React.FC<AdminButtonProps> = ({
    adminUrl,
    ...otherProps
}) => {
    if (adminUrl) {
        return <a href={adminUrl}><ArrowCircleRightIcon color="primary" {...otherProps}/></a>;
    }
    return null;
};
