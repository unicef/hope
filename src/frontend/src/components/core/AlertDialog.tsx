import Button from '@mui/material/Button';
import Dialog from '@mui/material/Dialog';
import DialogActions from '@mui/material/DialogActions';
import DialogContent from '@mui/material/DialogContent';
import DialogContentText from '@mui/material/DialogContentText';
import DialogTitle from '@mui/material/DialogTitle';
import * as React from 'react';
import { useEffect } from 'react';
import { useTranslation } from 'react-i18next';

export function AlertDialog({
  message,
  show,
}: {
  message: string;
  show: boolean;
}): React.ReactElement {
  const { t } = useTranslation();
  const [open, setOpen] = React.useState(false);
  const [oldShow, setOldShow] = React.useState(false);
  useEffect(() => {
    if (!oldShow && show) {
      setOpen(true);
    }
    setOldShow(show);
    // eslint-disable-next-line react-hooks/exhaustive-deps
  }, [show]);

  const handleClose = (): void => {
    setOpen(false);
  };

  return (
    <div>
      <Dialog
        open={open}
        onClose={handleClose}
        aria-labelledby="alert-dialog-title"
        aria-describedby="alert-dialog-description"
      >
        <DialogTitle id="alert-dialog-title">{t('Error')}</DialogTitle>
        <DialogContent>
          <DialogContentText id="alert-dialog-description">
            {message}
          </DialogContentText>
        </DialogContent>
        <DialogActions>
          <Button onClick={handleClose} color="primary" autoFocus>
            OK
          </Button>
        </DialogActions>
      </Dialog>
    </div>
  );
}
