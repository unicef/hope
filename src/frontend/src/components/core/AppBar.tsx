import { BusinessAreaSelect } from '@containers/BusinessAreaSelect';
import { GlobalProgramSelect } from '@containers/GlobalProgramSelect';
import { UserProfileMenu } from '@containers/UserProfileMenu';
import { useCachedMe } from '@hooks/useCachedMe';
import MenuIcon from '@mui/icons-material/Menu';
import TextsmsIcon from '@mui/icons-material/Textsms';
import { Box, Button } from '@mui/material';
import MuiAppBar from '@mui/material/AppBar';
import IconButton from '@mui/material/IconButton';
import Toolbar from '@mui/material/Toolbar';
import { styled } from '@mui/system';
import * as React from 'react';
import { MiśTheme, theme } from 'src/theme';

const StyledToolbar = styled(Toolbar)(() => ({
  display: 'flex',
  justifyContent: 'space-between',
}));

const StyledLink = styled('a')({
  textDecoration: 'none',
  color: '#e3e6e7',
});

interface AppBarProps {
  open: boolean;
}

interface StyledAppBarProps extends AppBarProps {
  theme: MiśTheme;
}
const StyledAppBar = styled(MuiAppBar)<StyledAppBarProps>(
  ({ theme: muiTheme, open }) => ({
    position: 'fixed',
    top: 0,
    zIndex: muiTheme.zIndex.drawer + 1,
    backgroundColor: muiTheme.palette.secondary.main,
    transition: muiTheme.transitions.create(['width', 'margin'], {
      easing: muiTheme.transitions.easing.sharp,
      duration: muiTheme.transitions.duration.leavingScreen,
    }),
    ...(open && {
      marginLeft: '270px',
      width: 'calc(100% - 270px)',
      transition: muiTheme.transitions.create(['width', 'margin'], {
        easing: muiTheme.transitions.easing.sharp,
        duration: muiTheme.transitions.duration.enteringScreen,
      }),
    }),
    ...(!open && {
      marginLeft: '0',
      width: '100%',
      transition: muiTheme.transitions.create(['width', 'margin'], {
        easing: muiTheme.transitions.easing.sharp,
        duration: muiTheme.transitions.duration.leavingScreen,
      }),
    }),
  }),
);

const StyledIconButton = styled(IconButton)<AppBarProps>(({ open }) => ({
  marginRight: 36,
  ...(open && {
    display: 'none',
  }),
}));

export const AppBar = ({ open, handleDrawerOpen }): React.ReactElement => {
  const { data: meData, loading: meLoading } = useCachedMe();
  const servicenow = `https://unicef.service-now.com/cc?id=sc_cat_item&sys_id=762ae3128747d91021cb670a0cbb35a7&HOPE - ${
    window.location.pathname.split('/')[2]
  }&Workspace: ${window.location.pathname.split('/')[1]} \n Url: ${
    window.location.href
  }`;

  if (meLoading) {
    return null;
  }
  return (
    <StyledAppBar theme={theme} open={open}>
      <StyledToolbar>
        <Box display="flex" alignItems="center" justifyContent="center">
          <Box ml={1}>
            <StyledIconButton
              edge="start"
              color="inherit"
              aria-label="open drawer"
              onClick={handleDrawerOpen}
              open={open}
            >
              <MenuIcon />
            </StyledIconButton>
          </Box>
          <Box display="flex" alignItems="center">
            <Box ml={6} data-cy="business-area-container">
              <BusinessAreaSelect />
            </Box>
            <Box ml={6} data-cy="global-program-filter-container">
              <GlobalProgramSelect />
            </Box>
          </Box>
        </Box>
        <Box display="flex" justifyContent="flex-end">
          <Button startIcon={<TextsmsIcon style={{ color: '#e3e6e7' }} />}>
            <StyledLink target="_blank" href={servicenow}>
              Support
            </StyledLink>
          </Button>
          <UserProfileMenu meData={meData} />
        </Box>
      </StyledToolbar>
    </StyledAppBar>
  );
};
