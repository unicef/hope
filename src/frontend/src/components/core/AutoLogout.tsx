import * as React from 'react';
import { useCallback, useEffect } from 'react';
import { useIdleTimer } from 'react-idle-timer';
import { AUTO_LOGOUT_MILLIS } from '../../config';

export function AutoLogout(): React.ReactElement {
  const handleOnIdle = (): void => {
    if (!localStorage.getItem('AUTHENTICATED')) {
      return;
    }
    window.location.assign('/api/logout');
    localStorage.removeItem('AUTHENTICATED');
  };

  //eslint-disable-next-line
  const { reset } = useIdleTimer({
    timeout: AUTO_LOGOUT_MILLIS,
    onIdle: handleOnIdle,
    debounce: 500,
  });

  const handleOnAction = useCallback((): void => {
    reset();
  }, [reset]);

  useEffect(() => {
    window.addEventListener('click', handleOnAction);
    window.addEventListener('mousemove', handleOnAction);

    return () => {
      window.removeEventListener('click', handleOnAction);
      window.removeEventListener('mousemove', handleOnAction);
    };
  }, [handleOnAction]);

  return <></>;
}
