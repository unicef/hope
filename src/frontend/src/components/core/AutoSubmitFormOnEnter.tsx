import { useFormikContext } from 'formik';
import * as React from 'react';
import { useEffect } from 'react';

export function AutoSubmitFormOnEnter(): React.ReactElement {
  const { submitForm } = useFormikContext();
  useEffect(() => {
    const handleEnter = (e: KeyboardEvent): void => {
      if (e.key === 'Enter') {
        submitForm();
      }
    };
    window.addEventListener('keypress', handleEnter);
    return () => {
      window.removeEventListener('keypress', handleEnter);
    };
  }, [submitForm]);
  return <></>;
}
