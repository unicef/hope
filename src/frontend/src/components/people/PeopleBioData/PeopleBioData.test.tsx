import { act } from 'react';
import wait from 'waait';
import { MockedProvider } from '@apollo/react-testing';
import { render } from '../../../testUtils/testUtils';
import { fakeHouseholdChoices } from '../../../../fixtures/population/fakeHouseholdChoices';
import { fakeIndividual } from '../../../../fixtures/population/fakeIndividual';
import { fakeApolloAllGrievances } from '../../../../fixtures/grievances/fakeApolloAllGrievances';
import { fakeGrievancesChoices } from '../../../../fixtures/grievances/fakeGrievancesChoices';
import { PeopleBioData } from './PeopleBioData';
import { fakeBaseUrl } from '../../../../fixtures/core/fakeBaseUrl';

describe('components/population/IndividualBioData', () => {
  it('should render', async () => {
    const { container } = render(
      <MockedProvider addTypename={false} mocks={fakeApolloAllGrievances}>
        <PeopleBioData
          baseUrl={fakeBaseUrl}
          businessArea="afghanistan"
          individual={fakeIndividual}
          choicesData={fakeHouseholdChoices}
          grievancesChoices={fakeGrievancesChoices}
        />
      </MockedProvider>,
    );
    await act(() => wait(0)); // wait for response
    expect(container).toMatchSnapshot();
  });
});
