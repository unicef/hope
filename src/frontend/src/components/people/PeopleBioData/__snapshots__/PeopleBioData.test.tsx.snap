// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/population/IndividualBioData should render 1`] = `
<div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-eOzmre jYcykM css-1ps6pg7-MuiPaper-root"
  >
    <div
      class="sc-egkSDF flCGEE"
    >
      <h6
        class="MuiTypography-root MuiTypography-h6 css-1j53xw6-MuiTypography-root"
      >
        Bio Data
      </h6>
    </div>
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-6 css-zow5z4-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Full Name
          </span>
          <div
            data-cy="label-Full Name"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              Jan Romaniak
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Given Name
          </span>
          <div
            data-cy="label-Given Name"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              Jan
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Middle Name
          </span>
          <div
            data-cy="label-Middle Name"
          >
            -
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Family Name
          </span>
          <div
            data-cy="label-Family Name"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              Romaniak
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Gender
          </span>
          <div
            data-cy="label-Gender"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              Male
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Age
          </span>
          <div
            data-cy="label-Age"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              121
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Date of Birth
          </span>
          <div
            data-cy="label-Date of Birth"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              <time
                datetime="-2176675200000"
              >
                10 Jan 1901
              </time>
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Estimated Date of Birth
          </span>
          <div
            data-cy="label-Estimated Date of Birth"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              No
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Marital Status
          </span>
          <div
            data-cy="label-Marital Status"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              Married
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Work Status
          </span>
          <div
            data-cy="label-Work Status"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              Not provided
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Pregnant
          </span>
          <div
            data-cy="label-Pregnant"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              No
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Role
          </span>
          <div
            data-cy="label-Role"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              None
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Preferred language
          </span>
          <div
            data-cy="label-Preferred language"
          >
            -
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Residence Status
          </span>
          <div
            data-cy="label-Residence Status"
          >
            -
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Country
          </span>
          <div
            data-cy="label-Country"
          >
            -
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Country of Origin
          </span>
          <div
            data-cy="label-Country of Origin"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              San Marino
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Address
          </span>
          <div
            data-cy="label-Address"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              938 Luna Cliffs Apt. 551
Jameschester, SC 24934
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Vilage
          </span>
          <div
            data-cy="label-Vilage"
          >
            -
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Zip Code
          </span>
          <div
            data-cy="label-Zip Code"
          >
            -
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Administrative Level 1
          </span>
          <div
            data-cy="label-Administrative Level 1"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              Ghazni
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Administrative Level 2
          </span>
          <div
            data-cy="label-Administrative Level 2"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              Abband
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Administrative Level 3
          </span>
          <div
            data-cy="label-Administrative Level 3"
          >
            -
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Administrative Level 4
          </span>
          <div
            data-cy="label-Administrative Level 4"
          >
            -
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 css-1osj8n2-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Geolocation
          </span>
          <div
            data-cy="label-Geolocation"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              -
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Data Collecting Type
          </span>
          <div
            data-cy="label-Data Collecting Type"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              data collecting type
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1idn90j-MuiGrid-root"
      >
        <div
          class="sc-eauhAA glRDYa"
        />
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Observed disabilities
          </span>
          <div
            data-cy="label-Observed disabilities"
          >
            -
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Seeing disability severity
          </span>
          <div
            data-cy="label-Seeing disability severity"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              None
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Hearing disability severity
          </span>
          <div
            data-cy="label-Hearing disability severity"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              None
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Physical disability severity
          </span>
          <div
            data-cy="label-Physical disability severity"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              None
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Remembering or concentrating disability severity
          </span>
          <div
            data-cy="label-Remembering or concentrating disability severity"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              None
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Self-care disability severity
          </span>
          <div
            data-cy="label-Self-care disability severity"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              None
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Communicating disability severity
          </span>
          <div
            data-cy="label-Communicating disability severity"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              None
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Disability
          </span>
          <div
            data-cy="label-Disability"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              Not Disabled
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1idn90j-MuiGrid-root"
      >
        <div
          class="sc-eauhAA glRDYa"
        />
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div
          class="MuiBox-root css-cgq59l"
        >
          <div
            class="MuiBox-root css-1itv5e3"
          >
            <div>
              <span
                class="sc-blHHSb dUfUKp"
                color="textSecondary"
              >
                National ID
              </span>
              <div
                data-cy="label-National ID"
              >
                <span
                  class="sc-gtLWhw dynQwi"
                  color="textSecondary"
                >
                  WSH221365
                </span>
              </div>
            </div>
          </div>
          <div>
            <span
              class="sc-blHHSb dUfUKp"
              color="textSecondary"
            >
              issued
            </span>
            <div
              data-cy="label-issued"
            >
              <span
                class="sc-gtLWhw dynQwi"
                color="textSecondary"
              >
                Poland
              </span>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1idn90j-MuiGrid-root"
      >
        <div
          class="sc-eauhAA glRDYa"
        />
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Email
          </span>
          <div
            data-cy="label-Email"
          >
            -
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Phone Number
          </span>
          <div
            data-cy="label-Phone Number"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              +48734123423
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Alternative Phone Number
          </span>
          <div
            data-cy="label-Alternative Phone Number"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              -
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1idn90j-MuiGrid-root"
      >
        <div
          class="sc-eauhAA glRDYa"
        />
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Date of last screening against sanctions list
          </span>
          <div
            data-cy="label-Date of last screening against sanctions list"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              <time
                datetime="1644305739718"
              >
                8 Feb 2022
              </time>
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 css-1osj8n2-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Linked Grievances
          </span>
          <div
            data-cy="label-Linked Grievances"
          >
            <span
              class="sc-gtLWhw dynQwi"
              color="textSecondary"
            >
              <span>
                -
              </span>
            </span>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1idn90j-MuiGrid-root"
      >
        <div
          class="sc-eauhAA glRDYa"
        />
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-gj1fbr-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Wallet Name
          </span>
          <div
            data-cy="label-Wallet Name"
          >
            -
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-gj1fbr-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Blockchain Name
          </span>
          <div
            data-cy="label-Blockchain Name"
          >
            -
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-gj1fbr-MuiGrid-root"
      >
        <div>
          <span
            class="sc-blHHSb dUfUKp"
            color="textSecondary"
          >
            Wallet Address
          </span>
          <div
            data-cy="label-Wallet Address"
          >
            -
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
